components/sidenav/index.css

```diff
@@ -221,6 +221,7 @@
 	margin-block-end: var(--mod-sidenav-heading-bottom-margin, var(--spectrum-sidenav-heading-bottom-margin));
 	padding-inline: var(--mod-sidenav-inline-padding, var(--spectrum-sidenav-inline-padding));
 	color: var(--mod-sidenav-header-color, var(--spectrum-sidenav-header-color));
+	font-family: var(--mod-sidenav-header-font-family, var(--spectrum-sidenav-header-font-family));
```

Comment:
@cdransf No need to add a new --mod here since that will add to our supported API. If a customer requires it down the road, we can address the need then.

Reply:
Sounds good! Just dropped it. 🙂